web/server/codechecker_server/api/report_server.py

```diff
@@ -1419,7 +1419,9 @@ def getDiffResultsHash(self, run_ids, report_hashes, diff_type,
 
                 base_hashes = session.query(Report.bug_id.label('bug_id')) \
                     .outerjoin(File, Report.file_id == File.id) \
-                    .filter(Report.detection_status.notin_(skip_statuses_str))
+                    .filter(
+                        Report.detection_status.notin_(skip_statuses_str),
+                        Report.fixed_at.is_(None))
```

Comment:
:exclamation: :exclamation: :exclamation: Fow now it is possible to specify review status and detection status filters in the CLI diff command. Do we wan't to allow this? :exclamation: :exclamation: :exclamation: 

Lets suppose that my bug is in the following state:
- Detection status: Resolved
- Review status: Confirmed
- Fix date: empty.

Steps:
- Storage 1:
```cpp
int main() {
  return sizeof(42);
}
```
- Storage 2:
```cpp
int main() {
  return 0;
}
```
- Change review status to confirmed on the UI.

Now let's suppose that I want to get resolved reports from the UI: `CodeChecker cmd diff -b myrun -n ./reports --resolved`. It will show me no results, because detection status filter has a default value.

Now if I run the following command: `CodeChecker cmd diff -b myrun -n ./reports --resolved --detection-status resolved` it will show me one report.